data/hosts.yml

```diff
@@ -766,6 +766,13 @@ hosts:
       - ip: 82.165.230.43
         domain: search.gmx.com
 
+  - name: Goodreads
+    items:
+      - ip: 52.94.237.126
+        domains:
+          - www.goodreads.com
+          - ads.goodreads.com
```

Comment:
Per #48 please refrain from using one IP for multiple domains, especially if they are likely to be loaded together on a single web page.